src/ekat/ekat_workspace_impl.hpp

```diff
@@ -152,6 +152,25 @@ WorkspaceManager<T, D>::get_space_in_slot(const int team_idx, const int slot) co
   return space;
 }
 
+template <typename T, typename D>
+template <typename S>
+KOKKOS_FORCEINLINE_FUNCTION
+Unmanaged<typename WorkspaceManager<T, D>::template view_1d<S> >
+WorkspaceManager<T, D>::get_n_spaces_in_slot(const int team_idx, const int slot, const int n) const
+{
+  Unmanaged<view_1d<S> > space(
+    reinterpret_cast<S*>(&m_data(team_idx, slot*m_total) + m_reserve),
+    n*(sizeof(T) == sizeof(S) ?
```

Comment:
do not want n* here

Reply:
Check the parentheses: sneaky @tcclevenger has wrapped the whole ?: ternary op in the parentheses, I think.

Reply:
Yeah, It looks ugly though. I changed it.

Reply:
Ah, my bad.